app/models/concerns/translated_content.rb

```diff
@@ -3,7 +3,7 @@ module TranslatedContent
   include RoleControl::ParentalControlled
 
   included do
-    has_paper_trail skip: [:langauge]
+    has_paper_trail skip: [:language]
```

Comment:
This typo fix doesn't seem to fix the bug, anyway, and it'd feel weird to revert to an incorrect spelling. I'll create an issue for the versioning bug.